src/pages/Analysis/Analysis.tsx

```diff
@@ -71,19 +151,31 @@ export const Analysis = () => {
       <div className={styles.tagChartBox}>
         <div className={styles.chartTitleBox}>
           <h2 className={styles.chartTitle}>자주 썼던 태그</h2>
-          <p className={styles.chartPeriod}>2023.10.09 ~ 2023.10.16</p>
+          <div className={styles.chartPeriodContainer}>
+            <p className={styles.chartPeriod}>
+              {startDate?.getFullYear()}년{' '}
+              {startDate?.getMonth() !== undefined && startDate?.getMonth() + 1}
+              월 {startDate?.getDate()}일
+            </p>
+            <p className={styles.chartPeriod}>~</p>
+            <p className={styles.chartPeriod}>
+              {endDate?.getFullYear()}년{' '}
+              {endDate?.getMonth() !== undefined && endDate.getMonth() + 1}월{' '}
+              {endDate?.getDate()}일
+            </p>
```

Comment:
원래 이렇게 하는 게 국룰인지는 모르겠지만
parsing 로직이 view를 구성하는 곳에 들어가 있어서 보기가 조금 어렵네여
차라리 parsing 로직을 String을 리턴하는 함수로 빼서 함수의 리턴값만 중괄호 안에 넣는 건 어떨까요

```suggestion
            <p className={styles.chartPeriod}>
              {getParsedDateString(startDate)}
            </p>
            <p className={styles.chartPeriod}>~</p>
            <p className={styles.chartPeriod}>
              {getParsedDateString(endDate)}
            </p>
```

Reply:
대화상대 api 연결할 때 수정해보겠습니다